### PR TITLE
NRI PR - Progress Update

### PR DESCRIPTION
**Pull request by helenziyihuang**
In this PR, we:
- Update our first progress update .md file

### CODE REVIEW

**Comment by syoh** [comment]
@helenziyihuang Could you turn on GitHub pages on the repository? The instruction video shows you how: https://ucsb-ds-capstone-2021.github.io/class/software-tools_create-update-pages.html

**Comment by helenziyihuang** [comment]
Turned on!

**Comment by syoh** [comment]
Great! I updated the Google doc: https://docs.google.com/document/d/1aLhpKz-epGHEaMbtc2jyWQiu0EQF9irU-w-QD2aWH4o/edit?ts=6018b3c2

**Comment by helenziyihuang** [comment]
Thank you.

File: ucsb_ds_capstone_projects_2021/projects/nri/update1.md [hunk 1]
```diff
@@ -1,2 +1,140 @@
...
+https://cran.r-project.org/web/packages/kinship2/index.html
+
+The purpose of KINSHIP2 is to create structures signifying correlations between family relationships (pedigrees). 
+
+
+## Goals
+
+Due to the Founder effect found in the various families, we can try to figure out background information on the anscestors of the family groups. 
+
+**First goal:**
```
Comment (matthewbcoleman) [inline ucsb_ds_capstone_projects_2021/projects/nri/update1.md]: I really liked your goals, they we're very descriptive

File: ucsb_ds_capstone_projects_2021/projects/nri/update1.md [hunk 2]
```diff
@@ -1,2 +1,140 @@
...
+The project seeks to examine a variant of Alzheimer's disease passed genetically and determine how far back the mutation/mutations responsible for said disorder trace back -- identifying the timeframe when this occurred can provide us key insights into whether or not the mutation was the result of potential selective advantage.
+
+
+## Background
+
+Listed below are the terminology and biology concepts that are fundamental to our project.
+
+**Inheritance of Genetic Variation**
+* Identical by descent (IBD) - IBD length is correlated with historical relationship
+* Identical by state (IBS)
```
Comment (matthewbcoleman) [inline ucsb_ds_capstone_projects_2021/projects/nri/update1.md:19]: Possible description of this?

Comment (matthewbcoleman) [inline ucsb_ds_capstone_projects_2021/projects/nri/update1.md:41]: Maybe put a description for IBS?

File: ucsb_ds_capstone_projects_2021/projects/nri/update1.md [hunk 3]
```diff
@@ -1,2 +1,140 @@
...
+* IBD2: shared 2 alleles
+* The length of the shared IBD segments is approximately exponentially distributed with a mean of 100 m^(-1) cM
+
+**Trio Phasing (Haplotype)**
+* Problem - We do not have genomes from both parents of every individual of interest
+* Solutions
+  - Option 1: Add more relatives into the phasing algorithm
+  - Option 2: Use known haplotypes from human populations and change your approach to a supervised method
+
+## Data Files
```
Comment (matthewbcoleman) [inline ucsb_ds_capstone_projects_2021/projects/nri/update1.md:92]: Really liked that you put the different files and put the descriptions about whether they were human readable or not

File: ucsb_ds_capstone_projects_2021/projects/nri/update1.md [hunk 4]
```diff
@@ -1,2 +1,140 @@
...
+
+PLINK is a command-line tool that is open-source and is essentially a whole-genome association analysis toolset desgined to perform basic, large-scale analyses on genetics.
+
+3. KINSHIP2
+https://cran.r-project.org/web/packages/kinship2/index.html
+
+The purpose of KINSHIP2 is to create structures signifying correlations between family relationships (pedigrees). 
+
+
+## Goals
```
Comment (matthewbcoleman) [inline ucsb_ds_capstone_projects_2021/projects/nri/update1.md:108]: I thought your goals were very clear and descriptive

**Pull request merged**